cmd/main.go

```diff
@@ -61,7 +73,7 @@ func main() {
 
 	// Discord
 	dcFormatter := formatter.NewFormatter()
-	botService := bot.NewManager(summaryService, reservationRepo).WithFormatter(dcFormatter).WithLogger(log)
+	botService := bot.NewManager(summaryService, reservationRepo, onlineChecker).WithFormatter(dcFormatter).WithLogger(log)
```

Comment:
The real question is if we want our infrastructure layer to "drive" online checker. 

In hexagon, we have layers: 

* we have "drivers" (basically - user input) that drives the "core" layer (our business)
* core (business) depends on some of the infrastructure, that is also driven, but this time by the "core" layer

My initial thought process is: 

* we have a driver, that is a discord bot
* this discord bot requires two general business domains: a summary feature, and a reservation system

Online checker implements "online" visibility to the summary service. Hence, what I would do, is I would pass `onlineChecker` as a dependency to `summaryService`. This `summaryService` then can include `isOnline` flag in the response.